pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/con_user-roles-in-pantheon.adoc

```diff
@@ -0,0 +1,20 @@
+[id="user-roles-in-pantheon_{context}"]
+= User roles in {ProductShortName}
+
+In {ProductShortName}, each user is assigned a role that determines the actions the user can perform.
```

Comment:
In general, use 2nd person. Suggest:
`Each {ProductShortName} user is assigned a role that determines what actions you can perform.`

Reply:
done

Reply:
changed to "As a Pantheon user, you are assigned a role that determines what actions you can perform."